index.html

```diff
@@ -0,0 +1,297 @@
+
+        <details id="q2_details">
+          <summary>Follow-up: time-limited UI</summary>
+          <fieldset>
+            <legend>
+              If you answered YES to question 1, is the interface only
+              available for a given amount of time?
+            </legend>
+            <div id="q2Description">
+              <p><em>For example:</em></p>
+              <ul>
+                <li>a notification that disappears</li>
+              </ul>
+              <p>
+                <strong>Why?</strong> There are a range of reasons why a
+                time-limited visual interface may cause problems for users;
+                for example, screen reader users may not be aware that the
+                interface is available (or else struggle with too many
+                interruptions), or users may not be able to reach the
+                interface in time to interact with it before it disappears.
+              </p>
+            </div>
+            <input type="radio" id="q2_yes" name="q2" value="Yes" aria-describedby="q2Description" />
+            <label for="q2_yes">Yes</label>
+            <input type="radio" id="q2_no" name="q2" value="No" aria-describedby="q2Description" />
+            <label for="q2_no">No</label>
+            <input type="radio" id="q2_na" name="q2" value="N/A" aria-describedby="q2Description" />
+            <label for="q2_na">N/A</label>
+          </fieldset>
+        </details>
+      </fieldset>
+
+      <fieldset>
+        <legend>
+          Does the design being proposed add anything audible to the user
+          interface?
+        </legend>
+        <div id="q3Description">
+          <p><em>For example:</em></p>
+          <ul>
+            <li>a notification sound</li>
+            <li>a speech-based interface</li>
+            <li>an audio stream (alone or with other timed media)</li>
+          </ul>
+          <p>
+            <strong>Why?</strong> Audio cues may not always be accessible to
+            users due to disability or because the use of audio is not
+            appropriate for them. Depending on the purpose of the audio, it
+            may be appropriate to provide mechanisms to provide alternative
+            forms for the same information, to provide a redundant visual
+            cue, or allow configuration options.
+          </p>
+        </div>
+        <input type="radio" id="q3_yes" name="q3" value="Yes" required aria-describedby="q3Description" />
+        <label for="q3_yes">Yes</label>
+        <input type="radio" id="q3_no" name="q3" value="No" required aria-describedby="q3Description" />
+        <label for="q3_no">No</label>
+        <input type="radio" id="q3_n/a" name="q3" value="N/A" required aria-describedby="q3Description" />
+        <label for="q3_n/a">N/A</label>
+      </fieldset>
+
+      <fieldset>
+        <legend>
+          Is any user-presented data passed through a protocol if the spec
+          is API-only?
```

Comment:
This seems like 2 separate questions (and I reordered the second):
1. Is the spec API-only?
   1. If so: Is any data passed through a protocol and then presented to a user?

But ... maybe the second question applies whether or not the spec includes non-API components? So maybe ...

```suggestion
          Does the spec define a protocol that conveys any data meant to be presented to a human?
```